eclipse-scout-core/src/desktop/outline/pages/PageWithTable.ts

```diff
@@ -288,8 +288,10 @@ export class PageWithTable extends Page implements PageWithTableModel {
   }
 
   protected _getLimitedResultInfoDo(tableData: any): LimitedResultInfoContributionDo {
-    return arrays.ensure(tableData?._contributions)
-      .find((contribution: DoEntity) => contribution._type === 'scout.LimitedResultInfoContribution') as LimitedResultInfoContributionDo;
+    if (!tableData) {
+      return null;
+    }
+    return dataObjects.getContribution('scout.LimitedResultInfoContribution', tableData) as LimitedResultInfoContributionDo;
```

Comment:
Maybe it would be easier, if dataObjects.getContribution would itself check if tableData is null?